generators/app/templates/src/app/components/SearchBar/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+interface Props {
+  buttonClassName?: string;
+  className?: string;
+  formClassName?: string;
+  textButtonSearch?: string;
+  children?: React.ReactNode;
+  handleSubmit?: (event: React.FormEvent<HTMLFormElement>) => void;
```

Comment:
In the previous implementation, this prop was required. Are we sure it isn't required anymore?

Reply:
Yeah I think it should be required, I don't see why a searchbar shouldn't have a way to submit